client/src/components/History/Modals/SelectorModal.vue

```diff
@@ -84,6 +89,8 @@ export default {
         this.fields = [
             { key: "name", sortable: true },
             { key: "tags", sortable: true },
+            { key: "count", label: "Length", sortable: true },
```

Comment:
Would probably call this "Count",  or actually even better, simply "Datasets" as the header?

Reply:
It's a good point.  In the old histories list we refer to it like this:

![image](https://user-images.githubusercontent.com/155398/187813471-aad648c6-f746-40e3-a41a-754ff34c887b.png)

Where 'Items' is what you're counting here.   Maybe we go with that for clarity?

Reply:
Ok, I'll go with `Items`